packages/components/src/components/icon-with-message/icon-with-message.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import classNames from 'classnames';
+import { isDesktop } from '@deriv/shared';
+import Text from '../text/text';
+import Div100vhContainer from '../div100vh-container';
+
+type TIconWithMessage = {
+    className?: string;
+    full_width?: boolean;
+    icon: React.ReactElement;
+    message: React.ReactNode;
+    text?: string | React.ReactElement;
+};
+
+const IconWithMessage = ({
+    children,
+    className,
+    full_width,
+    icon,
+    message,
+    text,
+}: React.PropsWithChildren<TIconWithMessage>) => (
+    <Div100vhContainer
+        className={classNames('account-management__message-wrapper', {
+            'account-management__message-wrapper-full-width': full_width,
+        })}
+        is_disabled={isDesktop()}
+        height_offset='110px'
+    >
+        <div
+            className={classNames('account-management__message-content', {
+                [`${className}__message-content`]: className,
+            })}
+        >
+            {icon && (
+                <div
+                    className={classNames('account-management__message-icon', {
+                        [`${className}__message-icon`]: className,
+                    })}
+                >
+                    {icon}
```

Comment:
would it be better to use here Icon component and just pass the name and size for this icon via props?